### DISCUSSION TITLE
Solution to Reward Contributors

### DISCUSSION THREAD

**Discussion by MehmetAdemi** [community/community#52044] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I’m happy to see that GitHub is further developing the GitHub Sponsors. However, as the latest blog post describes, there is still a long way to go in creating a financially sustainable solution for OSS and its contributors.  We are an organization that is source-available. Our community members contribute regularly, and we put bounties/rewards on certain issues to incentivize them to work with us, but also to appreciate their contributions. We’ve been using [Algora.io](http://algora.io/) as a service to do so. I would love to see a similar integration within GitHub itself. In the best case, working together with Algora, since they have the expertise, and infrastructure and are very committed to the subject.

**Comment by anjuan** [comment]
Thanks for your feedback! We’re always looking to our maintainers for suggestion and we’ve added this onto our backlog to look into.
